up42/workflow.py

```diff
@@ -694,3 +721,12 @@ def delete(self) -> None:
         self.auth._request(request_type="DELETE", url=url, return_text=False)
         logger.info(f"Successfully deleted workflow: {self.workflow_id}")
         del self
+
+    @property
+    def max_concurrent_jobs(self) -> int:
+        """Gets the maximum number of concurrent jobs allowed by the project settings."""
+        url = f"{self.auth._endpoint()}/projects/{self.project_id}/settings"
+        response_json = self.auth._request(request_type="GET", url=url)
+        project_settings = response_json["data"]
+        project_settings_dict = {d["name"]: int(d["value"]) for d in project_settings}
+        return project_settings_dict["MAX_CONCURRENT_JOBS"]
```

Comment:
Added this duplication to avoid circular imports from Project. We will need to fix later potentially with a ProjectSettings object.